src/ape/managers/accounts.py

```diff
@@ -20,17 +21,39 @@ def __repr__(self) -> str:
         accounts_str = ", ".join([a.address for a in self.accounts])
         return f"[{accounts_str}]"
 
-    @property
-    def containers(self) -> Dict[str, TestAccountContainerAPI]:
+    def __init__(self):
+        self.init_containers()
```

Comment:
Are we missing any `super` calls here?

Reply:
no.  this class is an exception and there is no super for this one